src/main/java/uk/gov/hmcts/reform/sscs/utility/dwpResponseUtil.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.sscs.utility;
+
+public class dwpResponseUtil {
+
+    public static final int MAX_DWP_RESPONSE_DAYS = 28;
```

Comment:
I think this might have changed to 31 days, based on this https://tools.hmcts.net/jira/browse/SSCSCI-831